main.cpp

```diff
@@ -139,18 +142,18 @@ float Car::calculateTrueSellValue(std::string carMake, float carSellingPrice, in
     return std::abs(trueSellValue);
 }
 
-int Car::calculateMilesUntilOilChange(int fuelTypeCar, int totalMiles)
+void Car::calculateMilesUntilOilChange(int fuelTypeCar, int totalMiles)
 {
     int milesUntilOilChange = 0;
 
     if (!(milesUntilOilChange = totalMiles % 5000))
     {
         std::cout << fuelTypeCar << " car needs doesn't need oil change" << std::endl;
-        return milesUntilOilChange;
+        return;
     }
 
     std::cout << "Car needs oil change in " << milesUntilOilChange << " miles"<< std::endl;
-    return milesUntilOilChange;                                       
+    return;                                     
```

Comment:
No need for this return statement at the end of this void function
- [ ] revise